### DISCUSSION TITLE
Welcome to UE4-Binary-Builder Discussions!

### DISCUSSION THREAD

**Discussion by ryanjon2040** [ryanjon2040/UE4-Binary-Builder#29] [General]
## 👋 Welcome to Unreal Binary Builder!
  Discussions is a place to connect with other members who use UE4 Binary Builder. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ARC345** [comment]
Does anyone have a linux binary built that i can just install?

**Reply by ryanjon2040** [reply]
I do have plans to write an app for Linux and Mac. Probably using electron.

**Comment by chaosrau** [comment]
How do you create a source code build with Server/Client build configurations. Even with the Include Server/ Include Client flags set, upon creating a project with the source version using this tool, there's no option to build dedicated server.

<img width="123" alt="image" src="https://user-images.githubusercontent.com/853826/168802106-c9f878af-0747-4238-a273-53032cf99c90.png">

For example as explained in official documentation

https://docs.unrealengine.com/4.26/en-US/InteractiveExperiences/Networking/HowTo/DedicatedServers/

I have also created the *Server.Build.cs as described. The Server project gets created as part of the solution, but there's no build configuration for building the server.

Generating the Unreal solution using batch files supplied enables this.

<img width="135" alt="image" src="https://user-images.githubusercontent.com/853826/168803854-8f6f2f2a-25b8-4d7c-8fb6-7745622487c3.png">
